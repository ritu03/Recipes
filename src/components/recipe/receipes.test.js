import React from 'react'
import { render, cleanup  } from '@testing-library/react'
import configureMockStore from 'redux-mock-store'
import Recipes from './recipes'

const mockStore = configureMockStore()
const storeValues = {
  recipeData: {
    data: null,
    status: ''
  }
}
const store = mockStore(storeValues)
describe('Recipes snapshot', () => {
  afterEach(cleanup)

  test('renders properly', () => {
    const { container } = render(<Recipes store={store} />)
    expect(container).toMatchSnapshot()
  })
  })