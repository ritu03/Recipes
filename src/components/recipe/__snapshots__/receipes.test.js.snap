// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipes snapshot renders properly 1`] = `
<div>
  <div
    class="sc-hKFxyN hHNCRy"
  >
    <header
      class="sc-dlnjwi JlHph"
    >
      <h1>
        Healthy Recipes by Ritu
      </h1>
    </header>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-8"
      />
    </div>
  </div>
</div>
`;
